src/dmd/mars.d

```diff
@@ -923,6 +925,55 @@ private int tryMain(size_t argc, const(char)** argv)
             cgFile.write();
         }
     }
+
+    static if (SYMBOL_TRACE)
+    {
+        // this is debug code we simply hope that we will not need more
+        // then 32 Megabyte of log-buffer;
+        char* fileBuffer = cast(char*)malloc(ushort.max * 512 * 64);
+        char* bufferPos = fileBuffer;
+
+        char[255] fileNameBuffer;
+        import core.stdc.time : ctime, time;
+        auto now = time(null);
+        sprintf(&fileNameBuffer[0],
+             "%s-%s.%s".ptr, "symbol".ptr, ctime(&now), "log".ptr);
+        auto f = File(&fileNameBuffer[0]);
```

Comment:
Your filename contains spaces and unprintable character '\n'
```
-        import core.stdc.time : ctime, time;
-        auto now = time(null);
-        sprintf(&fileNameBuffer[0],
-             "%s-%s.%s".ptr, "symbol".ptr, ctime(&now), "log".ptr);
+
+        import std.datetime : Clock;
+        auto dateTime = Clock.currTime();
+        auto dateStr = dateTime.toISOExtString();
+
+        sprintf(&fileNameBuffer[0], "%s-%s.%s".ptr, "symbol".ptr, dateStr.ptr, "log".ptr);
```
Would make it look like: 'symbol-2018-02-14T00:19:39.28761.log'